src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -113,6 +123,12 @@ private void CheckCandidate(DataCache snapshot, ECPoint pubkey, CandidateState c
             }
         }
 
+        /// <summary>
+        /// Determine whether the committee will be refreshed at the specified height.
```

Comment:
Hard code defined as once per size of the set of committee group.

Reply:
Updated.